JumpscaleLibs/sal/zosv2/ZOSv2.py

```diff
@@ -63,7 +63,13 @@ def reservation_create(self):
         return self._explorer.reservations.new()
 
     def reservation_register(
-        self, reservation, expiration_date, identity=None, expiration_provisioning=None, customer_tid=None
+        self,
+        reservation,
+        expiration_date,
+        identity=None,
+        expiration_provisioning=None,
+        customer_tid=None,
+        currencies=["TFT"],
```

Comment:
FreeTFT should be included here

Reply:
okay, will have it `["TFT","FreeTFT"] ` instead